argusim/run_job.py

```diff
@@ -60,21 +54,6 @@
 WHITE = "\033[37m"
 RESET = "\033[0m"  # Resets all attributes
 
-
-def build_simulator():
-    process = subprocess.Popen(
-        [
-            pose_simulator_build_script_abs,
-        ],
-        cwd=os.path.dirname(pose_simulator_build_script_abs),
-    )
-    while process.poll() is None:  # wait while the code is being built
-        time.sleep(0.1)
-    return_code = process.returncode
-    if return_code != 0:
-        raise AssertionError(RED + f"Build failed with code {return_code}" + RESET)
-
-
```

Comment:
why aren't we building the simulator anymore?

Reply:
It isn't great to couple building and running the sim. I can write another 'debug.sh' that first builds and then runs if that'll help. The point is though, I want to keep building away from run_job

Reply:
Added a run_debug.sh script that build sim before running monte carlo